system/modules/core/classes/Theme.php

```diff
@@ -548,8 +546,8 @@ protected function extractThemeFiles($arrFiles, $arrDbFields)
 							}
 						}
 
-						// Adjust the file paths in style sheets
-						elseif (\Config::get('uploadPath') != 'files' && ($table == 'tl_style_sheet' || $table == 'tl_style') && strpos($value, 'files') !== false)
+						// Adjust the file paths in style sheets and tl_files
+						elseif (\Config::get('uploadPath') != 'files' && ($table == 'tl_style_sheet' || $table == 'tl_style' || ($table == 'tl_files' && $name == 'path')) && strpos($value, 'files') !== false)
```

Comment:
IMO we should only change fields we know which data is in it. Just because a column value starts with `files/` doesn't ensure that it's a path that should be changed.